### PR TITLE
Cockpit 152

### PR DESCRIPTION
**Pull request by martinpitt**
Release notes. 152 has not finished releasing yet, but there won't be changes to the features/fixes concerning the release notes.

Rendered result can be seen here: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-152.html

### CODE REVIEW

**Comment by martinpitt** [comment]
@mvollmer, I'd appreciate a review from you particularly for the Apps section.

File: _posts/2017-10-06-cockpit-152.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 152.
+
+### Add Applications page
+
+This version introduces the "Applications" tool page. This is an extension
+point for third-party software to add themselves to the Cockpit interface, by
+shipping [AppStream](https://www.freedesktop.org/wiki/Distributions/AppStream/)
+metadata that contains a Cockpit manifest.
```
Comment (mvollmer) [inline _posts/2017-10-06-cockpit-152.md]: Strictly speaking, the AppStream metadata doesn't contain the Cockpit manifest.  "by shipping appropriate AppStream metadata".

Comment (martinpitt) [reply _posts/2017-10-06-cockpit-152.md]: Indeed, the AppStream data just contains the `<launchable type="cockpit-manifest">` tag, not the manifest itself. Adjusted, thanks!

**Comment by stefwalter** [comment]
The release notes look good. It seems like the Applications stuff should be talked about as future work. It's not "done" in the Cockpit sense of the word.

**Pull request merged**